ci/docker/runtime_functions.sh

```diff
@@ -1532,6 +1532,13 @@ nightly_estimator() {
     nosetests test_sentiment_rnn.py
 }
 
+nightly_global_numpy_flag() {
```

Comment:
Is it necessary to define a new run? Can you make it part of the existing nightly test suite instead?

Reply:
I put it into estimator nightly test

Reply:
Ok, so then there's no need to add this here, right? Or maybe you want to elaborate your intention :)

Reply:
Although putting in estimator together is not ideal, as long as the test is run alone, it's fine